src/RoleObjects/FormData.test.js

```diff
@@ -0,0 +1,160 @@
+import * as createjs from 'createjs-module';
+import AccessibilityModule from '../index';
+
+describe('FormData', () => {
+  describe('register role', () => {
+    let canvasEl;
+    let parentEl;
+    let stage;
+    let container;
+    let cjsForm;
+    let formEl;
+    let actionVal;
+    let shouldAutoComplete;
+    let charSetVal;
+    let enctypeVal;
+    let methodVal;
+    let nameVal;
+    let targetVal;
+
+    beforeEach(() => {
+      canvasEl = document.createElement('canvas');
+      parentEl = document.createElement('div');
+      stage = new createjs.Stage(canvasEl);
+      container = new createjs.Container();
+      cjsForm = new createjs.Shape(); // dummy object
+      actionVal = 'formsubmit.php';
+      shouldAutoComplete = false;
+      charSetVal = 'utf-8';
+      enctypeVal = 'text/plain';
+      methodVal = 'get';
+      nameVal = 'form_name';
+      targetVal = '_blank';
+
+      AccessibilityModule.register({
+        displayObject: container,
+        role: AccessibilityModule.ROLES.MAIN,
+      });
+      AccessibilityModule.setupStage(stage, parentEl);
+      stage.accessibilityTranslator.root = container;
+      stage.addChild(container);
+
+      AccessibilityModule.register({
+        accessibleOptions: {
+          action: actionVal,
+          autoComplete: shouldAutoComplete,
+          charSet: charSetVal,
+          enctype: enctypeVal,
+          method: methodVal,
+          name: nameVal,
+          target: targetVal,
+        },
+        displayObject: cjsForm,
+        parent: container,
+        role: AccessibilityModule.ROLES.FORM,
+      });
+
+      stage.accessibilityTranslator.update();
+    });
+
+    describe('rendering', () => {
+      it('creates form element', () => {
+        formEl = parentEl.querySelector('form');
+        expect(formEl).not.toBeNull();
+      });
+
+      it('sets \'action\' attribute', () => {
+        formEl = parentEl.querySelector(`form[action='${actionVal}']`);
+        expect(formEl).not.toBeNull();
+      });
+
+      it('sets \'autocomplete\' attribute', () => {
+        formEl = parentEl.querySelector('form[autocomplete]');
+        expect(formEl).not.toBeNull();
+      });
+
+      it('sets \'accept-charset\' attribute', () => {
+        formEl = parentEl.querySelector(`form[accept-charset='${charSetVal}']`);
+        expect(formEl).not.toBeNull();
+      });
```

Comment:
- interesting problem here (and this cropped up a few times in the work for this PR):
```Warning: Invalid DOM property `accept-charset`. Did you mean `acceptCharset`?```
I'm reticent to change the API to match what's expected, I'd rather go the route of fixing the internals and _deprecating_ the methods that would go away. Thoughts?